docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.9"
+
+services:
+  mongo:
+    image: mongo:latest
+    restart: unless-stopped
+    env_file: ./.env
+    ports:
+      - ${MONGO_LOCAL_PORT}:${MONGO_DOCKER_PORT}
+  app:
+    depends_on:
+      - mongo
+    image: ajax-onboarding-project
```

Comment:
you can set path to `Dockerfile`. So you don't need to build the image manually before starting the docker-compose.

see https://docs.docker.com/compose/compose-file/build/#dockerfile

Reply:
Done!